src/assets/specs/README.md

```diff
@@ -0,0 +1,29 @@
+## Generators
+
+### To generate go code:
+
+`oapi-codegen src/assets/specs/command-api.yaml > api/commands.go`
+
+#### Dependencies:
+
+* opai-codegen: To install:
+
+```bash
+go get github.com/deepmap/oapi-codegen/cmd/oapi-codegen
+```
+
+### To generate documentation:
+
+```
+mkdir tmp
+docker run --rm -v ${PWD}:/local openapitools/openapi-generator-cli generate -i /local/src/assets/specs/command-api.yaml -g html -o /local/tmp/
```

Comment:
Should the image to pinned to a version?

Reply:
I don't think so here. When we use a command like this to generate code, we should definitely insert a comment of the full command run, with a pinned version, that was used to generate it.

Reply:
That means that this command will break in a month or two when they push to `:latest` and the syntax changes, though?